cobigen-templates/templates-devon4j/src/main/templates/crud_openapi_angular_client_app/templates/src/app/${variables.etoName#lower_case}/${variables.etoName#lower_case}-grid/${variables.etoName#lower_case}-grid.component.ts.ftl

```diff
@@ -29,7 +29,7 @@ export class ${variables.etoName?cap_first}GridComponent implements OnInit {
         pageSize: 8,
         pageNumber: 0,
         sort: [{
-            property: '${model.properties[0].name!}',
+            <#if (model.properties[0].name)??> property: '${model.properties[0].name!}',</#if>
             direction: 'ASC'
         }]
```

Comment:
This will not work, you need to have an empty array in case the property is not set. Currently, you defined an potential object just defining direction if there is no property set. This will fail on server client communication.